esp/scm/ws_workunits_req_resp.ecm

```diff
@@ -1092,4 +1096,37 @@ ESPresponse [exceptions_inline] WUGetPluginsResponse
 {
     ESParray<ESPstruct WUEclPluginsInFolder> Plugins;
 };
+
+// ----------------------------------------------------------------------------------
+
+ESPrequest [nil_remove] WUAnalyseHotspotRequest
+{
+    string Wuid;
+    string RootScope;
```

Comment:
Is this optional? 
Should it default to the very root?

Reply:
yes it should be optional